src/universal/modules/userDashboard/containers/UserColumns/UserColumnsContainer.js

```diff
@@ -24,6 +24,11 @@ class UserColumnsContainer extends Component {
     }
   }
 
+  getProjectById = (projectId) =>
```

Comment:
this is a really neat pattern! probably only makes sense for drags, but it'd be interesting to see if we use it elsewhere.

Reply:
Than you! I was happy when the idea came to me!